com.ibm.streamsx.topology/opt/python/packages/streamsx/scripts/sc.py

```diff
@@ -19,10 +23,87 @@ def main(args=None):
     """
     cmd_args = _parse_args(args)
     topo = _create_topo(cmd_args)
+
+    # Get dependencies for app, if any at all
+    dependencies = _parse_dependencies()
+    # if dependencies and if -t arg, find & add local toolkits
+    if dependencies and cmd_args.spl_path:
+        tool_kits = cmd_args.spl_path.split(':')
+        # Check if any dependencies are in the passed in toolkits, if so add them
+        _add_local_toolkits(tool_kits, dependencies, topo)
+
     _add_toolkits(cmd_args, topo)
     _submit_build(cmd_args, topo)
     return 0
 
+def _parse_dependencies():
+    # Parse info.xml for the dependencies of the app you want to build sab file for
+    deps = {} # name - version
+    if os.path.exists('info.xml') and os.path.isfile('info.xml'):
```

Comment:
@Karsten12 Minor point but `os.path.exists` is redundant if checking with `os.path.isfile`